### ISSUE TITLE
Wrong Search Path

### ISSUE THREAD

**Issue by DavidRodriguezSoaresCUI** [qarmin/czkawka#371]
# History
I've used Czkawka 3.0.0 in the past to remove duplicate/similar images, with no problem. Today it doesn't work as expected.

# Current Behavior
I launch a "similar image search" on a folder A and the program actually tries to perform a search on another folder B.
![Czkawka_issue](https://user-images.githubusercontent.com/32097449/120103133-6a4d6900-c14e-11eb-9bee-b28bfaeb9b4d.png)
As you can see, I explicitely point to search directory on drive G: but there is actually activity on drive F:. I verified with Performance Monitor that it's actually Czkawka that is at the origin of the disk activity.

Note: Directory B is the location of my last "similar image search" from last time using the program a few months ago.
Note 2 : I tried both CLI and GUI, and Czkawka 3.0.0 and 3.1.0, with same problem .

# Expected behavior
Czkawka performs search on directory A, and activity can be seen on drive G:

# Context
Windows 64bit version 20H2
System with multiple physical drives

# Possible explanation
I suspect Czkawka stores somewhere the location(s) of last search, and somehow ignores CLI/GUI arguments when launching a new search.

**Comment by qarmin** [comment]
This is very strange and I never had got similar behavior.
Cli version doesn't store/load any settings so this is not a problem of remembering path.

Can you check from which disk will be printed duplicated files after ending of search?

I suspect that this may be OS problem, since I don't have any Idea how app could scan wrong disk.

**Comment by DavidRodriguezSoaresCUI** [comment]
I will let it end the search but this may take a few hours.

**Comment by DavidRodriguezSoaresCUI** [comment]
Update : As asked I ran the 3.1.0 CLI and let it scan to the end. Here are the results:
 - It took about 5 minutes, which is WAY less than expected because it was searching a folder with >100k image files. This indicates to me that it only enumerated them, not processed them.
 - The output file was where I expected : in the same directory from which I lauched the program.
 - After this, everything works as expected, the problem disappeared. Next time I ran the program it processes the directory I indicated.
 
Sorry for the anticlimactic ending but I guess this is a "black swan" event where a strange unexplained bug happens, and as a programmer I know how frustrating they are. Thank you for the assistance though 👍
